compiler/passes/buildDefaultFunctions.cpp

```diff
@@ -1820,18 +1822,18 @@ static void buildEnumStringOrBytesCastFunctions(EnumType* et,
   if (otherType != dtString && otherType != dtBytes) {
     INT_FATAL("wrong type was passed to buildEnumStringOrBytesCastFunctions");
   }
-  if (functionExists(astr_cast, otherType, et))
+  if (functionExists(astrScolon, otherType, et))
     return;
 
-  FnSymbol* fn = new FnSymbol(astr_cast);
+  FnSymbol* fn = new FnSymbol(astrScolon);
   fn->addFlag(FLAG_COMPILER_GENERATED);
   fn->addFlag(FLAG_LAST_RESORT);
```

Comment:
I think this and the other declaration should add FLAG_OPERATOR to the function as well